cmake/onnxruntime_providers.cmake

```diff
@@ -110,6 +110,9 @@ if (onnxruntime_USE_TENSORRT)
   add_definitions(-DUSE_TENSORRT=1)
   add_definitions("-DONNX_ML=1")
   add_definitions("-DONNX_NAMESPACE=onnx")
+  if (${onnxruntime_TENSORRT_MAX_BATCH_SIZE})
+    add_definitions(-DTENSORRT_MAX_BATCH_SIZE=${onnxruntime_TENSORRT_MAX_BATCH_SIZE})
```

Comment:
we want to be able to change the max_batch_size at runtime, not at build time.....

Reply:
done